gnome-initial-setup/gis-driver.c

```diff
@@ -80,12 +81,37 @@ struct _GisDriverPrivate {
   gchar *personality;
   gchar *lang_id;
 
+  gboolean is_live_session;
+
   GisDriverMode mode;
 };
 typedef struct _GisDriverPrivate GisDriverPrivate;
 
 G_DEFINE_TYPE_WITH_PRIVATE(GisDriver, gis_driver, GTK_TYPE_APPLICATION)
 
+static gboolean
+running_live_session (void)
```

Comment:
Why track this here as well? To skip the language page? Seems to be a no-op at the moment.